typescript-react/src/components/validation/index.ts

```diff
@@ -59,7 +63,7 @@ export const isNumberCreator = (message: string = 'Please enter a number') =>
     getValidatorErrorMessageOverride: message,
   });
 
-export const isIntegerCreator = (message: string = 'Must be a whole number') =>
+export const isIntegerCreator = (message: string = `Must be a whole number between ${INT_MIN_VALUE} and ${INT_MAX_VALUE}`) =>
```

Comment:
This message is highly confusing. No user understand what those limits mean :(
Usually you should have normal limits anyway. This is mostly just for fallback.
But change message to something more user friendly.

Must be a whole number with up to 10 digits